src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/VariantSummary.scss

```diff
@@ -55,9 +55,18 @@ For the reference (from Andrea)
   max-width: 60ch;
 }
 
-.clinicalSignificance {
-  display: flex;
-  flex-direction: column;
+.phenotypeAssociations {
+  margin-top: 20px;
```

Comment:
Is it intentional that the distance here is different from `.newRowGroup` class? If yes, could you confirm with Andrea that she wants the vertical space here to be different than, say, between "Ancestral" and "MAF", or between "GERP" and "Location."

Reply:
@azangru So as Andrea mentioned on Slack, this is fine?